routes/website/dashboard.router.js

```diff
@@ -12,6 +12,8 @@ const express = require('express'),
   relationsController = require('../../controllers/dashboard/relations.controller.js'),
   homesController = require('../../controllers/homes.controller.js'),
   emailsController = require('../../controllers/emails.controller.js'),
+  documentationsController = require('../../controllers/dashboard/documentations.controller.js'),
+  applicationController = require('../../controllers/application.controller.js'),
   flashHelper = require('../../helpers/flash.helper.js');
 
 const router = express.Router();
```

Comment:
This line needs to go 2 lines further down, below `module.exports = function (options) {`. This way, the application get a new router everytime you run the server (with appropriate configs etc). Otherwise, the previous router is re-used, which creates the weird errors you have in your tests.